src/sensors/SensorFusionRestDetect.h

```diff
@@ -0,0 +1,55 @@
+#ifndef SLIMEVR_SENSORFUSIONRESTDETECT_H_
+#define SLIMEVR_SENSORFUSIONRESTDETECT_H_
+
+#include "SensorFusion.h"
+
+#include "../motionprocessing/RestDetection.h"
+
+#if SENSOR_USE_VQF
+    #define SENSOR_FUSION_WITH_RESTDETECT 1
+#else
+    #define SENSOR_FUSION_WITH_RESTDETECT 0
+#endif
+
+namespace SlimeVR
+{
+    namespace Sensors
+    {
+        #if !SENSOR_WITH_REST_DETECT
+        struct SensorRestDetectionParams: RestDetectionParams {
+            SensorRestDetectionParams() : RestDetectionParams() {
+                restMinTimeMicros = 2.0f * 1e6;
+                restThGyr = 0.6f; // 400 norm
+                restThAcc = 0.06f; // 100 norm
+            }
+        };
+        #endif
+
+        class SensorFusionRestDetect : public SensorFusion
+        {
+        public:
+            SensorFusionRestDetect(float gyrTs, float accTs=-1.0, float magTs=-1.0)
+                : SensorFusion(gyrTs, accTs, magTs)
+            #if !SENSOR_WITH_REST_DETECT
+                , restDetection(restDetectionParams, gyrTs,
+                                ((accTs<0) ? gyrTs : accTs) )
```

Comment:
```suggestion
                                ((accTs<0) ? gyrTs : accTs))
```

Reply:
I might prefer to leave the space here, so that the double right bracelets won't be confused to be paring with the double left bracelets before the ? operation

Reply:
I guess or do it like this,
```suggestion
                                (accTs<0) ? gyrTs : accTs)
```

Reply:
That's less confusing as the first bracelet is clear to be in pairs so guess I'll take that. I've been used to guard every ? operator in its own bracelets and leave spaces in complex lines, but that one looks fairly simple.